srcpkgs/dhcp/template

```diff
@@ -35,7 +35,7 @@ pre_configure() {
 		--with-srv6-lease-file=/var/lib/dhcp/dhcpd6.leases
 		--with-cli-lease-file=/var/lib/dhclient/dhclient.leases
 		--with-cli6-lease-file=/var/lib/dhclient/dhclient6.leases
-		--with-randomdev=/dev/random"
+		--with-randomdev"
```

Comment:
I'm not sure what this gets us. Would it be a guarantee so an upstream change to `randomdev` is propagated?

Reply:
> Please follow the commit style: `dhcp: ....`

Done

> I'm not sure what this gets us. Would it be a guarantee so an upstream change to randomdev is propagated?

A tiny bit cleaner template file, and of course it doesn't guarantee what upstream does or doesn't do, but it's unlikely that the default value will change anytime soon (a better value would be `/dev/urandom` but then again we're not debating that).